Automattic/jetpack
# Round zoom level value in block controls

Fixes https://github.com/Automattic/wp-calypso/issues/74158

## Proposed changes:
This rounds the zoom level value shown in the Map block's controls. When zooming in/out of a Mapkit map, the camera distance gets converted to a Mapbox zoom level. This often contains lots of decimals.

This PR changes the rendering of that value in the block controls. This doesn't change the zoom value, it just changes the presentation.

<img width="299" alt="CleanShot 2023-03-07 at 13 56 43@2x" src="https://user-images.githubusercontent.com/528287/223428560-38cc99bb-730b-400c-9499-f829ad98234b.png">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
1. Apply this PR
2. Set the cookie to force the display of Mapkit maps, by pasting this in your console: `document.cookie = "map_provider=mapkit; path=/;";`
3. Add a post/page
4. Add a map block
5. Open the block properties to the right
6. Use the zoom buttons on the map
7. Changing the zoom should be reflected in the block's controls (might take tapping the button 2 times - because 1 zoom in Mapkit doesn't correspond to 1 zoom level in Mapbox).

It might feel counter intuitive that the value doesn't change every time you click the button, but this is a compromise.
